material.css

```diff
@@ -270,3 +270,24 @@ input.Button[type="submit"] {
     border-radius: 18px;
     margin-left: 6px;
 }
+
+.SessionDropdown .Dropdown-toggle .Avatar {
+    margin: 6px 5px -2px -6px;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+    font-size: 12px;
+    line-height: 24px;
+}
+
+@media (min-width: 768px)
+.NotificationsDropdown .Dropdown-toggle {
```

Comment:
Can you tell me what does this do?

Reply:
That may be connected to the below post. If it does cause issues remove it, and Ill find another fix for whatever the problem is. But, it seems to be maybe the Notification Icon.

Reply:
Okay 👍